utilities/install.py

```diff
@@ -170,6 +171,17 @@ def main(branch=branch, npmSkip=False):
     execute(cmd=['pip', 'install', '-e', '.', '--no-deps'])
 
 
+    # test
+    cprint("Testing")
```

Comment:
What about moving the tests before the npm stuff in order to speed up the process in case of fail? Also a flag to skip the tests on the install command would be nice

Reply:
I added a flag to skip tests (`--no-test`), and split the tests for pygeppetto and nwb-explorer.
I moved pygeppetto test right after pygeppetto installation and nwb-explorer test before `npm install` for webapp.